runtime.md

```diff
@@ -1,5 +1,51 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information is stored in `/run/oci`.
+The directory structure for a container is `<root>/<containerID>/state.json`.
+
+* **id** (string) ID is the container's ID.
```

Comment:
We need to add a comment that the reason we have an ID is because the hooks only get an open fd to this json document so they won't know the unique ID which may be useful for removal, etc.